node/src/services/builder.rs

```diff
@@ -163,9 +165,7 @@ where
     }
 }
 
-// TODO: Change this to RocksDB File Storage once it is implemented.
-impl<T> StorageLayerBuilder
-    for StorageHubBuilder<T, InMemoryFileStorage<T>, RocksDBForestStorage<T>>
+impl<T> StorageLayerBuilder for StorageHubBuilder<T, RocksDbFileStorage<T>, RocksDBForestStorage<T>>
```

Comment:
The In Memory implementation is there, above the RocksDB one. This change is because we previously didn't have the RocksDB impl for FileStorage and used the InMemory one as a placeholder.